cmds/core/shutdown/shutdown.go

```diff
@@ -33,21 +41,47 @@ var (
 		"-s":      unix.LINUX_REBOOT_CMD_SW_SUSPEND,
 	}
 	reboot = unix.Reboot
+	delay  = time.Sleep
 )
 
 func usage() {
 	log.Fatalf("shutdown [-h|-r|-s|halt|reboot|suspend] (defaults to halt)")
 }
 
 func main() {
-	if len(os.Args) == 1 {
-		os.Args = append(os.Args, "halt")
+	a := os.Args
```

Comment:
Shouldn't this be using pflags?

Reply:
no, the shutdown args don't follow any sane rules: a comand is  a switch or a not switch. So we don't use pflags.